samples/valueentity-counter/README.md

```diff
@@ -0,0 +1,85 @@
+# Implementing a Counter as a Value Entity
+
+This project is based on the Akka Serverless Maven archetype.
+
+```shell
+mvn archetype:generate \
+  -DarchetypeGroupId=com.lightbend \
+  -DarchetypeArtifactId=maven-archetype-akkasls \
+  -DarchetypeVersion=LATEST
+```
+
+See the [Kickstart a Maven project](https://developer.lightbend.com/docs/akka-serverless/java-services/kickstart.html) in the documentation for details.
+
+## Building and running unit tests
+
+To compile and test the code from the command line, use
+
+```shell
+mvn verify
+```
+
+## Running the integration tests
+
+The integration test sources are in the `src/it/java` directory.
+
+NOTE: As this is not part of the standard Maven setup, you might need to add it as test source directory in your IDE. (IntelliJ: right klick on the directory -> "Mark directory as" | "Test sources root")
+
+Maven is configured to run tests from it when using the `it` profile.
+
+```shell
+mvn verify -Pit
+```
+
+The integration test uses Docker via [TestContainers](https://www.testcontainers.org/) to set up the Akka Serverless environment before calling the gRPC API.
+
+## Running Locally
+
+In order to run your application locally, you must run the Akka Serverless proxy. The included `docker-compose` file contains the configuration required to run the proxy for a locally running application.
+It also contains the configuration to start a local Google Pub/Sub emulator that the Akka Serverless proxy will connect to.
+To start the proxy, run the following command from this directory:
+
+
+```shell
+docker-compose up
```

Comment:
```suggestion
docker compose up
```

Reply:
Is that new?
```
$ docker compose up
docker: 'compose' is not a docker command.
```
(on 20.10.2)

Reply:
I wonder if we should link to https://developer.lightbend.com/docs/akka-serverless/developing/running-service-locally.html instead of repeating that information here

Reply:
Looked into it and discovered https://github.com/lightbend/akkaserverless-docs/issues/621